includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -2108,9 +2108,9 @@ public static function print_status_meta_box( $post ) {
 							?>
 						</strong>
 						<?php if ( $is_amp_enabled && count( array_filter( $counts ) ) > 0 ) : ?>
-							<?php esc_html_e( 'AMP is enabled because no invalid markup is kept.', 'amp' ); ?>
+							<?php esc_html_e( 'AMP is enabled because no invalid markup has been kept.', 'amp' ); ?>
```

Comment:
Could also be:

```suggestion
							<?php esc_html_e( 'AMP is enabled because no invalid markup is being kept.', 'amp' ); ?>
```

But I don't feel strongly about either.